src/components/flame-graph/index.js

```diff
@@ -140,7 +140,7 @@ const options: ExplicitConnectOptions<{||}, StateProps, DispatchProps> = {
     const flameGraphTiming = selectedThreadSelectors.getFlameGraphTiming(state);
 
     return {
-      thread: selectedThreadSelectors.getThread(state),
+      thread: selectedThreadSelectors.getFilteredThread(state),
```

Comment:
Found this problem also when working on this PR. This caused the flame graph to break in a similar way when trying to hover transformed call stacks.

Reply:
This looks sensible, but can you please share a specific STR so that I try to reproduce locally too?

Reply:
Sure,

1. Load any profile in the flame graph.
2. Right-click a function belonging to a library, for instance `libxul.so`.
3. Choose "Collapse functions in libxul.so"

The new graph is incomplete, and the console shows this error:

```
Error: index undefined not in UniqueStringArray
unique-string-array.js:22:12
getString
unique-string-array.js:22:12
_drawCanvas
Canvas.js:280:25
_drawCanvas self-hosted:994:17 _scheduleDraw/</<
Canvas.js:66:10
timeCode
time-code.js:18:19
_scheduleDraw/<
```

4. Try hovering the boxes.

The flame graph disappears.

---

Let me know if you cannot reproduce and I can share a specific profile.

Reply:
Thanks, I could reproduce !